acceptance/utilities_test.go

```diff
@@ -123,9 +123,11 @@ func makeAppWithDir(appName string, instances int, deployFromCurrentDir bool, ap
 
 	// And check presence
 
-	out, err := Epinio("app list", "")
```

Comment:
oops :D !

Reply:
I had to move this into an Eventually block, it failed in Github actions with '1/2'. Worked locally, though.